tools/clang/include/clang/SPIRV/ModuleBuilder.h

```diff
@@ -66,6 +66,10 @@ class ModuleBuilder {
   inline void addExecutionMode(uint32_t entryPointId, spv::ExecutionMode em,
                                const std::vector<uint32_t> &params);
 
+  /// \brief Adds a debug name to the module under construction.
+  inline void addDebugName(uint32_t targetId, std::string name,
```

Comment:
Instead of having a `addDebugName()`, we can actually directly call `theModule.addDebugName()` in `BeginFunction()`, which should be extended to take an additional parameter for the function name. That way the debug name is grouped tightly with the function construction.